tests/test_sample_size/test_variables.py

```diff
@@ -0,0 +1,121 @@
+import unittest
+from unittest.mock import patch
+
+from statsmodels.stats.power import NormalIndPower
+from statsmodels.stats.power import TTestIndPower
+
+from sample_size.metrics import BooleanMetric
+from sample_size.metrics import NumericMetric
+from sample_size.metrics import RatioMetric
+
+
+class BooleanMetricTestCase(unittest.TestCase):
+    def setUp(self):
+        self.DEFAULT_MDE = 0.01
+        self.DEFAULT_PROBABILITY = 0.05
+        self.DEFAULT_MOCK_VALUE = 99
```

Comment:
```suggestion
        self.DEFAULT_VARIANCE = 99
```
? since it is mocked, so it's fine if the value is not the true "variance" I think